source/install.txt

```diff
@@ -10,168 +10,406 @@ Install Compass
+         Windows 7 or later and MongoDB 3.0 or later.
+
+         .. note::
+
+            The Windows installation of |compass| requires **Microsoft
+            .NET Framework version 4.5 or later.** The |compass-short|
+            installer prompts you to install the minimum required
+            version of the .NET framework if it is not already installed
+            on your system.
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+         |compass-short| requires the 64-bit version of Ubuntu 14.04 or
+         later and MongoDB 3.0 or later.
 
-Software requirements to run |compass-short| are listed below:
+     - id: rhel
+       name: RHEL
+       content: |
 
-.. include:: /includes/fact-compass-platforms.rst
+         |compass-short| requires the 64-bit version of RHEL 7+ or later
+         and MongoDB 3.0 or later.
 
-Installation
-------------
+Network Requirements
+~~~~~~~~~~~~~~~~~~~~
 
-.. note::
+Certain features of |compass|, such as third-party mapping services and
+Intercom, require inbound connections from external sources to
+function. In some instances, users have reported issues with their
+system firewall blocking these network requests and thereby limiting
+|compass| functionalities. To fix this issue, configure your system
+firewall to allow incoming connections for |compass|.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+Ensure ports ``80`` and ``443`` are open between |compass| and the
+host for the MongoDB Instance to which Compass connects.
+
+.. include:: /includes/fact-isolated-edition-firewall.rst
+
+Download Compass
+----------------
+
+You can use your web browser or the appropriate command line option to
+download |compass-short|.
+
+.. admonition:: |compass-short| bundled with MongoDB 3.6 and later
+   :class: important
+
+   |compass-short| is bundled with the MongoDB Server. If you choose 
+   to install |compass-short| when you install 
+   :dl:`MongoDB Server <enterprise>`, you can follow the tutorial in
+   the :manual:`MongoDB manual </reference/program/install_compass/>`
+   instead of this tutorial.
 
 .. tabs::
 
+   hidden: true
+
    tabs:
      - id: osx
-       name: OS X
+       name: macOS
        content: |
 
-         Open the {+download-page+} and download the latest version of
-         |compass| for OS X. The package downloads as a ``.dmg`` disk
-         image. Once you have downloaded |compass-short|, click on the
-         ``.dmg`` file to open the disk image within the OS X Finder.
-         Drag the |compass| application to your
-         :guilabel:`Applications` folder and unmount the disk image.
+         You can download |compass-short| using your browser or 
```

Comment:
Apologies if I missed this the first time, but I actually don't think we need both the per-OS sentences and the introductory sentence for this section of "You can use your web browser or the appropriate command line option to download Compass."

I might've missed that opening sentence in my prior review. I think it looks awkward having both so I think it's fine to remove the OS-specific sentences which state the same thing. Happy to defer to you or @gilberto-mdb if you feel differently.

Reply:
I think it makes sense to have at least one to provide the user with context on the tabs. I'd personally defer to keeping the first sentence "You can use your web browser or the appropriate command line option to download Compass." because it's more generic, and remove the OS-specific sentences like "You can download Compass using your browser or the macOS Terminal". @atsansone @jeff-allen-mongo

Reply:
Agreed ^

Reply:
Would have preferred to keep the OS-specific versions, but I have a way to handle this.